README.md

```diff
@@ -449,6 +449,71 @@ Note that because this method is not available in the generic `IDataReader` inte
 TimeSpan timeSpanTime = ((SnowflakeDbDataReader)reader).GetTimeSpan(13);
 ```
 
+Executing a Batch of SQL Statements (Multi-Statement Support)
+--------------------------------------------------------------
+
+With the .NET connector, you can send
+a [batch of SQL statements](https://docs.microsoft.com/en-us/sql/odbc/reference/develop-app/batches-of-sql-statements?view=sql-server-ver15)  (separated by semicolons)
+to be executed in a single request. For example:
+
+```cs
+IDbCommand cmd = conn.CreateCommand();
+                var param = cmd.CreateParameter();
+                param.ParameterName = "MULTI_STATEMENT_COUNT";
+                param.DbType = DbType.Int16;
+                param.Value = 3;
+                cmd.Parameters.Add(param);
+                cmd.CommandText = "select 1; select 2, 3; select 4, 5, 6";
+                IDataReader reader = cmd.ExecuteReader();
+```
+
+To send a batch of statements with the Snowflake .NET connector, you must specify the number of statements in the batch.
+The Snowflake database requires the exact number of statements in order to guard against
+[SQL injection](https://en.wikipedia.org/wiki/SQL_injection>) attacks. 
+
+The next section explains how to specify the number of statements in a batch. 
+
+### Specifying the Number of Statements in a Batch
+
+By default, the Snowflake database expects a .NET connector application to prepare and send a single statement for execution.
+
+You can override this by specifying the number of statements in a batch for a given request or by enabling multiple statements for
+the current session.
+
+To enable multiple statements for the current session or account, alter the session or account, and set the Snowflake
+  [MULTI_STATEMENT_COUNT <label-parameters__multi_statement_count>](https://docs.snowflake.com/en/sql-reference/parameters.html#label-parameters-multi-statement-count) parameter to ``0``.
```

Comment:
`<label-parameters__multi_statement_count>` should not be in the link text.

Reply:
Done